libs/escrow-hooks/src/index.ts

```diff
@@ -1,14 +1,10 @@
 export { default as useDebounce } from './useDebounce';
-export { default as useDeposit } from './useDeposit';
-export { default as useEscrowZap } from './useEscrowZap';
+export { default as useEscrowEvents } from './useEscrowEvents';
 export { default as useInvoiceDetails } from './useInvoiceDetails';
```

Comment:
useInvoiceDetails, useLock, useResolve, useTokenBalance are also hooks that you can directly from @smartinvoicexyz/hooks

Lets please remove anything that can reused and reduce the duplication as much as possible.

Reply:
Actually if the other hooks have nothing specific to DM, then we can move the hooks over to SI and then reuse here so we can get rid of the `escrow-hooks` package in this repo

Reply:
@dan13ram they are using details different details version.
I though about make details hooks called outside these hooks but SI are using them already so we will need to change SI front end code.
Or we can Do move them as they are to SI and add a version to them